src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -137,8 +125,25 @@ def get_instance_loader(*, class_: type) -> Callable:
     return partial(load_instance, class_=class_)
 
 
+@lru_cache(maxsize=None)
+def get_dict_of_loader(*, class_: type) -> Callable:
```

Comment:
```suggestion
def get_dict_of_instance_loader(*, class_: type) -> Callable:
```

There is `get_dict_of_arrays_loader` and `get_curve_dict_loader`. My bad using inconsistent names.